fanout-fanin/README.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: pattern
+title: FanOut-FanIn 
+folder: fanout-fanin
+permalink: /patterns/fanout-fanin/
+categories: Concurrency
+language: en
+tags:
+- Cloud distributed
+- Performance
+---
+
+## Intent
+The pattern that refers to executing multiple instances of the activity function concurrently. Generally "FanOut" 
+process is splitting the data into multiple chunks that would be passed to the activity function. "FanIn" process is 
+basically aggregating the result from individual instance of activity function and forming a single final result. <br />
+
+Each activity function is a long-running process that takes in a chunk of data. A callback function, or a Consumer 
+is passed to the activity function, that will be called when the long-running process gets over. The consumer will 
+process the individual return value from activity function as they are done running into a final result.
```

Comment:
The term activity function is not self-explanatory. Could we perhaps talk about microservices?

Reply:
We should also explain how the caller eventually gets the aggregated result.

Reply:
We need an explanation section and some of the details could be placed there.